packages/samples/headless-ssr/cypress/e2e/ssr-e2e-utils.ts

```diff
@@ -0,0 +1,17 @@
+export const ConsoleAliases = {
+  error: '@consoleError',
+  warn: '@consoleWarn',
+  log: '@consoleLog',
+};
+
+export function stubConsole() {
+  cy.window().then((win) => {
+    cy.stub(win.console, 'error').as(ConsoleAliases.error.substring(1));
```

Comment:
Wouldn't `cy.spy` be better as the msgs will still appear on console?

Reply:
yes, good point.